### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by BlesslinJerishR**
- Add Form
- Add Name
- Add Email
- Add Textarea
- Add Button
- Add Footer image
- Add Form Validation
- Add Formspree API

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -232,5 +232,47 @@ <h1 class="s">Skills</h1>
...
+          cols="30"
+          rows="10"
+          maxlength="500"
+          required
+          placeholder="
+        Hey I’d like to connect to..."
+        ></textarea>
+        <img class="foot" src="imgs/footer.png" alt="illustration" />
+        <br />
+        <button class="btn btn-2 btn-3" type="submit">Get in touch</button>
```
Comment (AdnanOlarmmi) [inline index.html:272]: - [ ] [OPTIONAL] Kindly note that the default type for a button is `submit`, so you can do away with that extra code 🙃 i.e. `type="submit"`.

File: index.html [hunk 2]
```diff
@@ -232,5 +232,47 @@ <h1 class="s">Skills</h1>
...
+      </h1>
+    </section>
+    <section class="center">
+      <form action="https://formspree.io/f/xeqwrjjn" method="POST">
+        <input
+          type="text"
+          class="in"
+          maxlength="30"
+          name="name"
+          placeholder="&nbsp;&nbsp;&nbsp;Full name"
```
Comment (AdnanOlarmmi) [inline index.html:250]: - [ ] [OPTIONAL] I think you could have used padding on the inputs for consistency sake, instead of this 😄

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @BlesslinJerishR,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Nice one following the design guidelines.
- You used formspree appropriately.

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BlesslinJerishR** [comment]
merge

**Pull request merged**